Cargo.toml

```diff
@@ -24,13 +24,15 @@ default = []
 
 
 [dependencies]
-
+accesskit_consumer = "0.24.0"
+accesskit = "0.16.0"
+parking_lot = "0.12"
 
 [dev-dependencies]
-
+egui = { version = "0.28.1", features = ["accesskit"] }
 
 [patch.crates-io]
-
+egui = { git = "https://github.com/emilk/egui"}
```

Comment:
You should specify a specific `rev` here in order to get predicable builds (important when for check we check out old revisions of `kittest`, e.g. for `git bisect`)

Reply:
Doesn't the Cargo.lock take care of that? 🤔

Reply:
Yeah, good point 👍